editor/blocks/text-block/index.js

```diff
@@ -1,11 +1,11 @@
-const { query, html } = wp.blocks.query;
+const { html } = wp.blocks.query;
 
 wp.blocks.registerBlock( 'wp/text', {
 	title: 'Text',
 	icon: 'text',
 
 	attributes: {
-		value: query( 'p', html() )
+		value: html()
```

Comment:
Might still be value in specifying `'p'` argument to at least limit value to text within the `<p>` element of the block?